docs/src/acb.md

```diff
@@ -726,19 +726,42 @@ lindep(A::Matrix{AcbFieldElem}, bits::Int)
 
 **Examples**
 
-```julia
-CC = AcbField(128)
-
-# These are two of the roots of x^5 + 3x + 1
-a = CC(1.0050669478588622428791051888364775253, - 0.93725915669289182697903585868761513585)
-b = CC(-0.33198902958450931620250069492231652319)
-
-# We recover the polynomial from one root....
-V1 = [CC(1), a, a^2, a^3, a^4, a^5];
-W = lindep(V1, 20)
-
-# ...or from two
-V2 = [CC(1), b, b^2, b^3, b^4, b^5];
-Vs = [V1 V2]
-X = lindep(Vs, 20)
+```jldoctest
+julia> CC = AcbField(128)
+Complex Field with 128 bits of precision and error bounds
+
+julia> # These are two of the roots of x^5 + 3x + 1
+
+julia> a = CC(1.0050669478588622428791051888364775253, -0.93725915669289182697903585868761513585)
+[1.00506694785886230292248910700436681509 +/- 1.80e-40] - [0.937259156692891837181491609953809529543 +/- 7.71e-41]*im
+
+julia> b = CC(-0.33198902958450931620250069492231652319)
+-[0.331989029584509320880414406929048709571 +/- 3.62e-40]
+
+julia> V1 = [CC(1), a, a^2, a^3, a^4, a^5]; # We recover the polynomial from one root....
+
+julia> W = lindep(V1, 20)
+6-element Vector{ZZRingElem}:
+ 1
+ 3
+ 0
+ 0
+ 0
+ 1
+
+julia> V2 = [CC(1), b, b^2, b^3, b^4, b^5]; # ...or from two
+
+julia> Vs = [V1 V2]
+6×2 Matrix{AcbFieldElem}:
+ 1.00000000000000000000000000000000000000                                                                               …  1.00000000000000000000000000000000000000
+ [1.00506694785886230292248910700436681509 +/- 1.80e-40] - [0.937259156692891837181491609953809529543 +/- 7.71e-41]*im     -[0.331989029584509320880414406929048709571 +/- 3.62e-40]
+ [0.131704842873658261608907475851071865533 +/- 2.43e-40] - [1.88401639994019194629934002026077386471 +/- 2.35e-39]*im     [0.110216715764464205102727554344054759368 +/- 3.32e-40]
+ [-1.63343943775826359832802955077858762779 +/- 7.18e-39] + [-2.0170041827680651335098567246278775869 +/- 5.08e-38]*im     [-0.0365907405106361618384680031506015710184 +/- 8.30e-41]
+ [-3.5321716296072262805776584101060548180 +/- 5.08e-38] + [-0.49626816785103656510289466486482917885 +/- 2.35e-39]*im     [0.0121477244339046924274232580429164920524 +/- 8.57e-41]
+ [-4.01520084357658781437077178965706076158 +/- 5.97e-39] + [2.81177747007867551213313071756269751169 +/- 3.99e-39]*im  …  [-0.00403291124647205167662794872826031818905 +/- 7.87e-42]
+
+julia> X = lindep(Vs, 20)
+2-element Vector{ZZRingElem}:
+ 0
+ 1
```

Comment:
Actually, I think something is wrong here: the input should perhaps be transposed? Indeed:
```
julia> lindep(Matrix(transpose(Vs)), 20)
6-element Vector{ZZRingElem}:
 1
 3
 0
 0
 0
 1
```
makes more sense to me, and also better fits with the docstring of `lindep`.

(So yeah, this example probably just was wrong for many years *sigh*)

Reply:
fixed